### PR TITLE
TE7-T1151: Add User Library Playlist

### PR DESCRIPTION
**Pull request by DevarshBhalara**
This PR Adds User Library Tab which include User's followed Playlist, Artists and Album

### CODE REVIEW

File: My Spotify/My Spotify/Coordinators/Storyboarded.swift [hunk 1]
```diff
@@ -10,6 +10,7 @@ import UIKit
 enum Storyboard: String {
     case auth = "Auth"
     case home = "Home"
+    case libraryUser = "UserLibrary"
```
Comment (Kruna1Pate1) [inline My Spotify/My Spotify/Coordinators/Storyboarded.swift]: make name same

File: My Spotify/My Spotify/Modules/UserLibrary/Cell/CategoryCell.swift [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+        let transparentView = UIView(frame: bounds.insetBy(dx: 0, dy:  -30))
+        transparentView.backgroundColor = #colorLiteral(red: 0, green: 0, blue: 0, alpha: 0)
+        backgroundView = transparentView
+
+        var frameView = frame
+        frameView.size.height -= 30
+        frameView.size.width -= 30
+        let blueView = UIView(frame: frameView)
+        blueView.backgroundColor = UIColor(named: "DarkGreen")
```
Comment (Kruna1Pate1) [inline My Spotify/My Spotify/Modules/UserLibrary/Cell/CategoryCell.swift]: blueView with name DarkGreen colour ?

File: My Spotify/My Spotify/Modules/UserLibrary/Cell/LibraryArtistsCell.swift [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    
+    // MARK: - Outlets
+    @IBOutlet private weak var lblArtistsName: UILabel!
+    @IBOutlet private weak var imgArtists: UIImageView!
+    
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+    override func layoutSubviews() {
```
Comment (Kruna1Pate1) [inline My Spotify/My Spotify/Modules/UserLibrary/Cell/LibraryArtistsCell.swift:21]: add line break

File: My Spotify/My Spotify/Modules/UserLibrary/ViewModel/LibraryViewModel.swift [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+            case .failure(let error):
+                self?.failure.value = error.body
+            }
+        }
+    }
+    
+    // MARK: - Get library playlist
+    func getLibraryPlaylist() {
+        APIManager.shared.call(type: .getUserPlaylists) { [weak self] (result: Result<LibraryPlaylist, CustomError>) in
+            switch result {
```
Comment (Kruna1Pate1) [inline My Spotify/My Spotify/Modules/UserLibrary/ViewModel/LibraryViewModel.swift:85]: use guard self

File: My Spotify/My Spotify/Network Layer/APIManager.swift [hunk 5]
```diff
@@ -51,6 +51,27 @@ class APIManager {
         }
     }
     
+    //MARK: - Load image from URL api
```
Comment (Kruna1Pate1) [inline My Spotify/My Spotify/Network Layer/APIManager.swift:54]: Where this method is used ?

**Review by Kruna1Pate1 — CHANGES_REQUESTED** [review]
Changes requested

**Review by Kruna1Pate1 — APPROVED** [review]

File: My Spotify/My Spotify/Network Layer/APIManager.swift (hunk 5 above)
Comment (DevarshBhalara) [reply My Spotify/My Spotify/Network Layer/APIManager.swift:54]: I didn't used but is came from develop from rebase

**Pull request merged**